examples/sphere/solidbody.jl

```diff
@@ -107,10 +66,125 @@ function linkfig(figpath, alt = "")
     end
 end
 
-linkfig("output/$(dirname)/error.png", "Absolute error in height")
+FT = Float64
+ne_seq = 2 .^ (2, 3, 4, 5)
+Δh = zeros(FT, length(ne_seq))
+L1err, L2err, Linferr = zeros(FT, length(ne_seq)),
+zeros(FT, length(ne_seq)),
+zeros(FT, length(ne_seq))
+Nq = 4
+
+# h-refinement study
+for (k, ne) in enumerate(ne_seq)
+    domain = Domains.SphereDomain(R)
+    mesh = Meshes.Mesh2D(domain, Meshes.EquiangularSphereWarp(), ne)
+    grid_topology = Topologies.Grid2DTopology(mesh)
+    quad = Spaces.Quadratures.GLL{Nq}()
+    space = Spaces.SpectralElementSpace2D(grid_topology, quad)
+
+    coords = Fields.coordinate_field(space)
+
+    Δh[k] = 2 * R / ne
+
+    h_init = map(coords) do coord
+        n_coord = Geometry.components(Geometry.Cartesian123Point(coord))
+        n_center = Geometry.components(Geometry.Cartesian123Point(center))
+        # https://en.wikipedia.org/wiki/Great-circle_distance
+        rd = R * atan(norm(n_coord × n_center), dot(n_coord, n_center))
+
+        if test_name == gaussian_test_name
+            h0 * exp(-(rd / r0)^2 / 2)
+        else # default test case, cosine bell
+            if rd < r0
+                h0 / 2 * (1 + cospi(rd / r0))
+            else
+                0.0
+            end
+        end
+    end
+
+    u = map(coords) do coord
+        ϕ = coord.lat
+        λ = coord.long
+
+        uu = u0 * (cosd(α0) * cosd(ϕ) + sind(α0) * cosd(λ) * sind(ϕ))
+        uv = -u0 * sind(α0) * sind(λ)
+        Geometry.UVVector(uu, uv)
+    end
+
+    function rhs!(dh, h, u, t)
+        div = Operators.Divergence()
+
+        @. dh = -div(h * u) # strong form of equation
+        Spaces.weighted_dss!(dh)
+    end
+
+    # Set the RHS function
+    rhs!(similar(h_init), h_init, u, 0.0)
+
+    # Solve the ODE
+    T = 86400 * 12
+    dt = 20 * 60
+    prob = ODEProblem(rhs!, h_init, (0.0, T), u)
+    sol = solve(
+        prob,
+        SSPRK33(),
+        dt = dt,
+        saveat = dt,
+        progress = true,
+        adaptive = false,
+        progress_message = (dt, u, p, t) -> t,
+    )
+    L1err[k] = norm(sol.u[end] .- h_init, 1)
+    L2err[k] = norm(sol.u[end] .- h_init)
+    Linferr[k] = norm(sol.u[end] .- h_init, Inf)
+
+    @info "Test case: $(test_name) with α: $(test_angle_name)"
+    @info "Number of elements per cube panel: $(ne) x $(ne)"
+    @info "Solution norm at t = 0: ", norm(h_init)
+    @info "Solution norm at t = $(T): ", norm(sol.u[end])
+    @info "L₁ error at t = $(T): ", L1err[k]
+    @info "L₂ error at t = $(T): ", L2err[k]
+    @info "L∞ error at t = $(T): ", Linferr[k]
+end
+
+# Plot the errors
+Plots.png(
+    Plots.plot(
+        1:length(ne_seq),
+        L1err,
+        xscale = :log,
+        yscale = :log,
+        xlabel = "Nₑ",
+        ylabel = "L₁ err",
+    ),
+    joinpath(path, "L1error.png"),
+)
+linkfig("output/$(dirname)/L1error.png", "L₁ error Vs Nₑ")
```

Comment:
```suggestion
linkfig(relpath(joinpath(path,"L1error.png"), "../.."), "L₁ error Vs Nₑ")
```
The DG one doesn't work either. The path should be relative to the top-level (ClimaCore.jl) directory.